docs/api-reference/geo-layers/mvt-layer.md

```diff
@@ -130,25 +130,22 @@ tileJSON (Object) - the tileJSON fetched
 
 ##### `onViewportChange` (Function, optional)
 
-`onViewportChange` called when the viewport changes or all tiles in the current viewport have been loaded. A function (`getRenderedFeatures`) to calculate the features rendered in the current viewport is passed as a JSON to this callback function.
-
+`onViewportChange` called when the viewport changes or all tiles in the current viewport have been loaded.
```

Comment:
Suggestion: since you are also exposing these functions as layer methods, the callback can just be called with `{layer: this}`, then the user can cal `layer.getVisibleTiles`.

Either way the new methods should be documented.